### PR TITLE
KOGITO-1716:  Show warning if user opens scesim file for rule based test scenario 

### PR DESCRIPTION
**Pull request by yesamer**
@danielezonca @jstastny-cz Can you please test and review it?

As you know, currently RULE Test Scenario, is not supported in Kogito (VSCode). 
The user can't create a new RULE Test Scenario, but he can loads a scesim file of a RULE Test scenario. 
With the changes introduced in this tiny PR, now a WARN type notification will be shown in case user loads a RULE Test Scenario

Screenshot from 2020-07-06 11-09-44

https://issues.redhat.com/browse/KOGITO-2060

### CODE REVIEW

**Comment by yesamer** [comment]
jenkins retest this please

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/events/ScenarioNotificationEvent.java [hunk 1]
```diff
@@ -27,11 +27,19 @@
...
+    private final boolean autoHide;
     private final org.uberfire.workbench.events.NotificationEvent.NotificationType notificationType;
 
     public ScenarioNotificationEvent(String message, NotificationEvent.NotificationType type) {
         this.message = message;
         this.notificationType = type;
+        this.autoHide = true;
+    }
+
+    public ScenarioNotificationEvent(String message, boolean autoHide, NotificationEvent.NotificationType type) {
```
Comment (jstastny-cz) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/events/ScenarioNotificationEvent.java:39]: I would maybe keep the additional argument (the one distinguishing the overloaded method) as the last one.

Comment (yesamer) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/events/ScenarioNotificationEvent.java]: @jstastny-cz Done.

**Comment by yesamer** [comment]
@kiegroup/gatekeepers can you please merge it?

**Pull request merged**